src/tools/U8U16Test/en.txt

```diff
@@ -0,0 +1 @@
+Steve Ballmer replaced Gates as CEO in 2000, and later envisioned a "devices and services" strategy. This unfolded with Microsoft acquiring Danger Inc. in 2008, entering the personal computer production market for the first time in June 2012 with the launch of the Microsoft Surface line of tablet computers, and later f
```

Comment:
Are we required to list attribution of this sample text? @DHowett-MSFT

Reply:
The sample text comes from the Wikipedia entries for Microsoft. I could put the words into a random order, maybe.

Reply:
I think that use still requires attribution per Wikipedia's licensing. If you just put the source URL and the date retrieved INTO the sample, I think we satisfy it and also it just adds a little more text to the sample.

Reply:
https://en.wikipedia.org/wiki/Wikipedia:Text_of_Creative_Commons_Attribution-ShareAlike_3.0_Unported_License is the license at the bottom of the page.

Attribution required under CC BY-SA 3.0. Even if you Share/Remix it. 

So I'd just do "Retrieved from https://en.wikipedia.org/wiki/Microsoft on 2020-01-17. Uses https://creativecommons.org/licenses/by-sa/3.0/ license."

Reply:
The other alternative, of course, is to use some sort of public domain text or a generator of junk/garbage text.

Reply:
I didn't expect this could be such a big issue. Even if I already considered to add the copyright I decided to omit it because it adds ASCII which influence the results for non-ASCII languages. However, we still have the original test results here 🙂